.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules.
+        run: yarn
+      - name: Build 'src'.
+        run: yarn build
+      - name: Build 'electron'.
+        run: yarn build-electron
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
This job could run on the previous docker. Also, why do we need to build the app first to run the linting?
I think you could just create the docker machine, run yarn and then run the linting and the prettier as well. :)

Reply:
Okay I'm gonna split it into 2 different job files entirely then. That is what you want right?

Reply:
No. I dont think you need to actions for that. You just need to remove the first one.

Reply:
I think this should work:

```
name: Linting

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
  workflow_dispatch:

jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Install modules.
        run: yarn
      - name: Lint code.
        run: yarn lint
```

Reply:
So no need to run build? I thought it would be nice to check for that as well.